Logger/LogFactory.cs

```diff
@@ -1,11 +1,33 @@
 ﻿namespace Logger
 {
-    public class LogFactory
+    using System;
+    using System.IO;
+
+    public static class LogFactory
     {
-        public BaseLogger CreateLogger(string className)
+        private static string? FilePath { get; set; }
+
+        public static FileLogger CreateLogger(string className)
         {
-            
-            return null;
+            FileLogger? fileLogger;
+            if (File.Exists(FilePath))
```

Comment:
```suggestion
            if (File.Exists(FilePath))
```
You may also want to check if FilePath is null & consider programmatically creating the file as mentioned in a previous comment of mine.